cli.go

```diff
@@ -117,7 +116,7 @@ func (i *CLI) Setup() (bool, error) {
 			if httpPort == "" {
 				httpPort = i.c.String("pprof-port")
 			}
-			httpAddr := fmt.Sprintf("localhost:%s", httpPort)
+			httpAddr := fmt.Sprintf("0.0.0.0:%s", httpPort)
```

Comment:
I would prefer a change to an explicit `"http-addr"` option, defaulting to `127.0.0.1:1627` (`'travis-ci/worker'.scan(/./).map(&:ord).sum`). Said another way, I don't think that the default should be for the API to serve on all interfaces.

Reply:
Okay, that seems reasonable.

Reply:
There may not be need for a default here: right now, passing a port is required to even enable this API in the first place. With this change, you would have to specify the whole address including the port, to enable the API. So we don't need a default address/port unless we start enabling the HTTP API by default. Should we?

Reply:
I handled this by removing the `pprof-port` and `http-api-port` config defs and replaced them both with `http-api-addr`. It really simplifies the logic here, which definitely confused me the first time I was trying to figure out what was needed to turn on the HTTP API.